hub/powertoys/run.md

```diff
@@ -106,9 +100,9 @@ The PowerToys Run settings menu includes a plugin manager that allows you to ena
 The Windows System Commands plugin provides a set of system level actions that can be executed.
 
 > [!TIP]
-> If your system language is supported by PowerToys, the system commands will be localized. If you prefer English commands, you can disable the setting **'Use localized system commands instead of English ones'**  in the [plugin manager](#plugin-manager).
+> If your system language is supported by PowerToys, the system commands will be localized. If you prefer English commands, you can disable the setting **Use localized system commands instead of English ones** in the [plugin manager](#plugin-manager).
```

Comment:
I added this signs to state that the word have to be seen together as the settings title.

Reply:
Oh I see. I use the bold text to adress UI elements. Same thing, I guess.

Reply:
I think we should use both. The sign to encapsulatevthe name parts and the bold format to highlight the settings title

Reply:
I think both methods try to do the same thing and it's double-up. We don't use the single quotes in any other place.